docs/user_guide/model_management.md

```diff
@@ -236,6 +236,11 @@ in-flight sequence(s) will be routed to the same model instance for processing.
 It is currently the responsibility of the user to ensure any in-flight
 sequence(s) are completed before reloading a sequence model.
 
+* Starting from r24.09, if the version policy on the model configuration is
```

Comment:
how difficult is it to provide an example? 

No fault here - but I don't quite follow the change / not sure others would understand. 

If it's already considered 'expected' behavior - maybe we can just update the descriptions below ...